### DISCUSSION TITLE
Full Screen Code View

### DISCUSSION THREAD

**Discussion by dhuang-hub** [community/community#3339] [Mobile]
It would be nice to be have a "read" mode feature for code views - e.g. full screen code display without the menu borders when the mobile is in a horizontal position.
Or a toggle setting for a "disappearing" menu borders, unless summoned with a tap near the screen borders. 
Full display for code display.

Thanks!

**Comment by candyho** [comment]
Hey, thanks for the feedback. We hear you. I'll create an internal issue to keep track of this request and will evaluate priority from time to time.
